Added `includesSome` to `DefaultFilterTypes` in Table (#1934)
iTwin/iTwinUI: .changeset/itchy-swans-jog.md

---
"@itwin/itwinui-react": patch
---

> Fixed the Table column `filter`'s type to include `"includesSome"`, which is a default filter function that we already supported.

Reviewer: Slight rephrase (emphasizing `Table` component, rewording "include", and removing "we").

```suggestion
Fixed types for `Table` column `filter` to allow `"includesSome"`, which is an already-supported filter function.
```
